src/Razor/src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -11,6 +11,8 @@
 
     <!-- Allow assemblies outside of lib in the package -->
     <NoWarn>$(NoWarn);NU5100</NoWarn>
+    <!-- It is intentional to not include build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets -->
```

Comment:
I'm adding an exclusion for a warning during pack. From what I can understand this is intentional. Is that correct @NTaylorMullen @ajaybhargavb?